webapp/sources/rudder/rudder-rest/src/test/resources/api/api_settings.yml

```diff
@@ -12,15 +12,13 @@ response:
         "settings":{
           "allowed_networks":[
             {
-              "id":"root",
-              "allowed_networks":[
-                "192.168.2.0/32"
-              ]
+              "id":"node-dsc",
+              "allowed_networks":[]
             },
             {
-              "id":"node-dsc",
+              "id":"root",
               "allowed_networks":[
-
+                "192.168.2.0/32"
               ]
```

Comment:
I'm not sure why that change is needed. I suspect it's not related to that commit but to the change with dsc/splaytime which, for some reason, changed the order of root/dsc node (it's just a flip between the two even if the diff doesn't make it clear)